doc/tutorials/outline.md

```diff
@@ -0,0 +1,88 @@
+# Tutorial Outline
+
+These guides already exists
+
+- [components](components.md)
+- [development](development.md)
+- [interactivity](interactivity.md)
+- [layouts](layouts.md)
+- [param](param.md)
+- [structure](structure.md)
+- [styling](styling.md)
+
+They should be refactored into the below
+
+## Beginner
+
+These guides are for you that is just starting to discover Panel.
+
+You will learn to develop and deploy amazing tools and apps that can be contained within a single Python file or notebook.
+
+- [ ] Getting installed (just a link to getting started)
+  - [ ] Using a pre-installed environment for this tutorial: Purpose is to lower Barrier of entry. Make it possible to do at large conference with poor internet. Use Binder, Github spaces, Anaconda Notebooks ???
+- [ ] [Serve Apps](https://github.com/holoviz/panel/blob/docs_fixes_1.4_a1_review/doc/tutorials/beginner/panel_serve.md)
```

Comment:
I've been thinking about it too. I think it should be judged on how the tutorial [Serve Panel Apps](https://github.com/holoviz/panel/blob/docs_fixes_1.4_a1_review/doc/tutorials/beginner/panel_serve.md) actually work both standalone and in the context of other guides.

Maybe its really just the title of the tutorial you want to change to *First Steps*, *Serve your first app*, *Say Hello to Panel* or similar?

## Pro

- Its exactly what `fastAPI` does. It just calls it *First Steps*. See https://fastapi.tiangolo.com/tutorial/first-steps/
- There are just basic things to explain about serving apps that if not explained our users (based on my experience) do not understand. And they get stuck.
- In addition a "Hello World" World Panel example is explained in great details. Which is important because Panels `.extension` and `.servable` concepts are a bit more complicated to understand than what competing frameworks have to offer.

## Con

- Is *Serve an App* engaging enough to catch users attention?
- Could the content be integrated in other guides and would it be more natural to have there.